widgets/mem_freebsd.lua

```diff
@@ -0,0 +1,67 @@
+----------------------------------------------
```

Comment:
I can have a look on it and add the swap code. That's good: don't know how the script is behaving when there is no swap available. Maybe it throws an error ...